### ISSUE TITLE
ref to UniBicocca

### ISSUE THREAD

**Issue by rjpbonnal** [andreareginato/conference_site#16]
Add reference in the main page/header to University of Milano Bicocca, Dipartimento di Informatica Sistemistica e Comunicazione with title :"in collaboration with...DISCO and AlgoLab" http://www.algolab.eu/ and logo of Uni http://www.disco.unimib.it/res/customer/css/default.gif

**Comment by andreaforni** [comment]
Hi,
I tried to put the text and the logo somewhere. I would like to do some tests to find the better position but tonight I did not have the time. 

I tried to put the text "in collaboration with...DISCO and AlgoLab" in the header, under "Spreading the Ruby and Rails way in Italy" but the logo takes too much space. 
We have some space between the "Call for paper" and the social media buttons but I've already seen with the "subscribe form" that is better to keep empty that space, otherwise the page become too heavy.

So, as first try, I put it in the right column, under the "mailing list panel" and above the "Golden sponsor":

![Near "In the right column"](https://github.com/andreaforni/conference_site/wiki/images/issues/university/uni_right_column.jpg)

 I do not know if it's the right place. The sponsors are pushed down in the page, in this way they have less visibility. This can be a problem? 

Other possible positions are: in the footer (adding a fourth column in first position) or in a "Credits" page, as I wrote in the mailing list. I can try these two solutions only tomorrow in the evening. Otherwise, if it's right the proposed solution I can do a pull request (I've already pushed the branch in my repository).

bye,
Andrea

**Comment by andreaforni** [comment]
I've just seen the screen shot I posted in the previous comment. Maybe we can move the "university panel" at the bottom of the column, under the "Media Sponsor" group. In this way the sponsor gain visibility.

**Comment by rjpbonnal** [comment]
Why not put the lodo etc... near (right) Milan 15 june 2012 ? it should fit without problems and have the right visibility.

**Comment by andreaforni** [comment]
Yeah, I'll make the change tomorrow, this evening I have no time...

**Comment by rjpbonnal** [comment]
also a reference to algo lab would be great

On Fri, Feb 17, 2012 at 12:09 PM, Andrea Forni <
reply@reply.github.com

> wrote:
> 
> Yeah, I'll make the change tomorrow, this evening I have no time...
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> 
> https://github.com/andreareginato/conference_site/issues/16#issuecomment-4018574

## 

Ra

**Comment by andreaforni** [comment]
Moved to rubydayit/conference_site#3. This issue can be closed.
